aws-lc-rs/README.md

```diff
@@ -70,6 +70,42 @@ the pre-generated bindings. This feature requires `libclang` to be installed. Se
 [requirements](https://rust-lang.github.io/rust-bindgen/requirements.html)
 for [rust-bindgen](https://github.com/rust-lang/rust-bindgen)
 
+##### prebuilt-nasm
+
+Enables the use of crate provided prebuilt NASM objects under certain conditions. This only affects builds for
+Windows x86-64 platforms. This feature is ignored if the "fips" feature is also enabled.
+
+Use of prebuilt NASM objects is prevented if either of the following conditions are true:
+* The NASM assembler is detected in the build environment
+* `AWS_LC_SYS_PREBUILT_NASM` environment variable is set with a value of `0`
+
+Be aware that [features are additive](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification);
+by enabling this feature, it is enabled for crates within the same build.
+
+## Use of prebuilt NASM objects
+
+For Windows x86 and x86-64, NASM is required for assembly code compilation. On these platforms,
```

Comment:
Does windows arm work with our arm assembly without nasm?

Reply:
For Windows/ARM we use the clang-cl (part of MSVC) which can handle assembly code, so there's no need for a separate assembler.